pkg/sources/git/git.go

```diff
@@ -1005,6 +1005,11 @@ func (s *Git) handleBinary(ctx context.Context, gitDir string, reporter sources.
 		return nil
 	}
 
+	if s.skipBinaries {
+		fileCtx.Logger().V(5).Info("skipping binary file", "path", path)
+		return nil
+	}
+
 	var handlerOpts []handlers.Option
 	if s.skipBinaries {
```

Comment:
can you delete this block now that it's a no-op?

Reply:
Probably. I'm just not sure of why it was done this way originally. Wanted check with @ahrav, but he is out today, and I was trying to get this fix in fast.

Reply:
i spoke with him about this yesterday and he said it was just an oversight, so i'm trying to not compound the damage :)

Reply:
I'm guessing that he just wanted to skip binaries and considered archives separately. I think the filtering is flawed though because it is based on file extension and unix binaries don't usually have one.

Reply:
oh, sorry, i'm not trying to resolve the question functionality in this pr. (it's a good question, i just wasn't trying to do that here.) my comment was: i'm pretty sure this block is dead, and i don't want to merge dead code. (maybe i'm just missing something that makes it non-dead though.)